src/drivers/driver.ts

```diff
@@ -549,6 +549,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
     return this.doRefreshExpansions(async () => {
       log.debug('Run _refreshExpansions cb');
       const opts = this.expansionOptions;
+      opts.envOverride = await this.getConfigureEnvironment();
```

Comment:
Should `.envOverride` be set as part of `this.expansionOptions`?  I think that's a bigger change but seems like if we do it here we should be doing it everywhere else too.

Reply:
I can't quickly answer this. Do you want to look at this more thoroughly for 1.8 and move the bug out of 1.7.1?  I am thinking we can also include this fix as is for 1.7.1, to unblock the main scenario we are aware of and revisit for 1.8 whether there is a better place to do this check?

Reply:
That proposal sounds good to me.  I'll approve this and then let's make sure we're tracking the full fix for 1.8.

Reply:
https://github.com/microsoft/vscode-cmake-tools/issues/1798